docs/src/main/asciidoc/_admin-guide/General_configuration_concepts.adoc

```diff
@@ -213,3 +213,40 @@ element, the value in a `host.xml` taking precedence over anything from
 `domain.xml`, and a value in a `<server-group/>` element taking
 precedence over a value specified directly under the `domain.xml` root
 element.
+
+== Script Configuration Files
+
+Scripts are located in the `$JBOSS_HOME/bin` directory. Within this directory you will find script configuration files
+for standalone and domain startup scripts for each platform. These files can be used to configure your environment
+without having to edit the scripts themselves. For example, you can configure the `JAVA_OPTS` environment variable to
+configure the JVM before the container is launched.
+
+=== Standalone Script Configuration Files:
+
+* `standalone.conf` invoked from `standalone.sh`
+* `standalone.conf.bat` invoked from `standalone.bat`
+* `standalone.conf.ps1` invoked from `standalone.ps1`
+
+=== Domain Script Configuration Files:
+
+* `domain.conf` invoked from `domain.sh`
+* `domain.conf.bat` invoked from `domain.bat`
+* `domain.conf.ps1` invoked from `domain.ps1`
+
+By default, these are in the `$JBOSS_HOME/bin` directory. However, you can set the `STANDALONE_CONF` environment
+variable for standalone servers or `DOMAIN_CONF` environment variable for domain servers with a value of the
+absolute path to the file.
+
+=== Common Script Configuration Files
+
+Starting with WildFly 23, common configuration files were introduced. These files are invoked from every script in the
+`$JBOSS_HOME/bin` directory. While these configuration files are not present in the directory by default, they
+can be added. You can simply just add the `common.conf` configuration file for the script type you want to
+execute and all scripts in the directory will invoke the configuration script.
+
+* `common.conf` for bash scripts
+* `common.conf.bat` for Windows batch scripts
+* `common.conf.ps1` for PowerShell scripts
+
+You can also set the `COMMON_CONF` environment variable to have this configuration script live outside the
+`$JBOSS_HOME/bin` directory.
```

Comment:
Possible additional paragraph:

Note that if you provide both a `common.conf` configuration file and a `standalone.conf` or `domain.conf` file, the standalone or domain startup scripts will call both files, first calling [whichever is first] and then calling [the other one].

Reply:
That makes sense to add. Thank you!